server/src/main/kotlin/org/javacs/kt/KotlinTextDocumentService.kt

```diff
@@ -254,27 +249,22 @@ class KotlinTextDocumentService(
 
     private fun lintLater(uri: URI) {
         lintTodo.add(uri)
-        if (!linting) {
```

Comment:
This means that if lintting is running, a new lintting won't be scheduled - but the already running linting may be running for an older version of the document's content. So better to always schedule the lintting (the doLint's implementation will avoid unnecessary recompiles anyway).